src/development/going-live.md

```diff
@@ -92,10 +133,46 @@ platform domain:add secure.example.com --cert=/etc/ssl/private/secure-example-co
 
 See `platform help domain:add` for more information.
 
+> **Success**
+> Everything should be fine and dandy. Congratulations. 
+> If not, well, we are here to support you, but first, you should try to figure-out by yourself what went wrong. Which is what the next section is about.
+
+## Initial Troubleshooting
+
+You did all of the above and when you visit your website at https://www.example.com (or whatever your real domain name) something is wrong? Here are some simple self-help steps you can do.
+
+### 1. Verify DNS
+On the command line with OS X or Linux (or using the Linux subsystem for Windows) type `host www.example.com`:
+
+The response should be something like
+```
+www.example.com is an alias for master-t2xxqeifuhpzg.eu.platform.sh.
+master-t2xxqeifuhpzg.eu.platform.sh has address 54.76.136.188
+```
+
+1. If it is not either you have not configured correctly your DNS server, or the DNS configuration did not propagate yet. As a first step you can try and remove your local DNS cache. 
+2. You can also try to set your DNS server to the Google public DNS server (8.8.8.8/8.8.4.4) to see if the issue is with the DNS server you are using.
+3. Try to run `ping www.example.com` (with you own domain name) if the result is different from what you got form the `host www.example.com` you might want to verify your `/etc/hosts` file (or its windows equivalent), you might have left there an entry from testing.
+
+### 2. Verify SSL
+
+On the command line with OS X or Linux (or using the Linux subsystem for Windows) type `curl -I -v  https://example.com` (again using your own domain):
+
+The response should be long. Look for error messages. They are usually explicit enough. Often the problem will be with a mismatch between the certificate and the domain name.
+
```

Comment:
Should we add a note that Lets Encrypt requires a redeploy after adding the domain name?